src/main/resources/assets/languages/de_DE.json

```diff
@@ -69,7 +69,11 @@
       "header": "%1$s%2$s (Ratelimited)\n **Du kannst den Befehl in %3$s wieder nutzen.**",
       "spam_1": "Mach bitte eine Pause, das wäre auch für deine Gesundheit gut :( *Beachte, dass das Ratelimit weiter erhöht wird, wenn du es, bevor es ausläuft, weiterversuchst!*",
       "spam_2": "\nIch denke aufhören ist jetzt die beste Option..."
-    }
+    },
+    "generic_error": "Wenn du willst, tritt unserem **Support Server** bei (Link auf `support.mantaro.site` zu finden), oder schau auf unserer GitHub Seite nach (`https://github.com/Kodehawa/MantaroBot`). Bitte sag denen, dass sie aufhören sollen mich hochzujagen und bitte vergiss nicht die Fehler-ID beim Melden eines Fehlers!",
+    "boom_quotes": ["Offenbar hat Megumin unser Schloss in die Luft gejagt...", "Uh-oh, offenbar hat unser Meister ein paar Nullen und Einsen auf dem Boden vergessen :<", "W-Warte, was ist gerade passiert?", "I-Ich denke hier brennt's... Du solltest vielleicht meinen Meister bitten, sich das anzuschauen.", "Ich habe Explosionsmagie gemeistert, siehst du?", "Vielleicht ist hier einfach etwas schiefgelaufen, aber, u-uh, Ich kann es nicht richtigstellen!", "U-Uhh.. Was wolltest du?"],
+    "ratelimit_quotes": ["Woah... you're calling me a bit too fast... I might get dizzy!", "Sei nicht gierig!", "Y-You're calling me so fast that I'm getting dizzy...", "Wart mal!", "Warte nur einen kleeiinen Augenblick länger uwu", "Schaut so aus, als würden wir noch ein Bußgeld bekommen, wenn wir so schnell weitermachen!", "Ich will das machen... aber warte noch einen Moment bitte.", "Hey, wart mal, Ich bin mit meiner Pause noch nicht fertig!", "Kannst du ein bisschen langsamer machen?"],
```

Comment:
`Woah... you're calling me a bit too fast... I might get dizzy!` can be translated to `Woah... Du rufts mich viel zu schnell... Mir wird ganz schwindlig.`
`Y-You're calling me so fast that I'm getting dizzy...` can be translated to `D-Du rufst much viel zu schnell.. mir ist schwindlig`

Reply:
Typo in "kleinen" (kleeiinen)

Reply:
On purpose original string was "tiiiiny" so changed it on purpose to that.
For the other two strings I had the same idea, though its meaning is somehow weird. Would take it though, if you think it's good enough.

Reply:
The first two strings work that way translated there is no real good way to translate them but it's better than leaving them in english. And for the longer tiny you could do "kleiiinen" similiar to how english does it but basically breaking the word entirely is kinda weird.

Reply:
Ok, guess I'll take this that way~